packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -27,12 +27,13 @@ const {
  * can be considered equivalents (without identity discontinuity).
  */
 const ESGlobalKeys = new Set([
-    // *** 18.1 Value Properties of the Global Object
+    // *** 19.1 Value Properties of the Global Object
+    'globalThis',
```

Comment:
debugging I noticed that `globalThis` was being remapped during the initialization, of course, the remapping of it means nothing because it was replaced with itself, basically unnecessary initialization step. I don't remember why it wasn't part of this list when we first compile it.